### PR TITLE
Add Pokes

### PR DESCRIPTION
**Pull request by ThePsionic**
Same as #662 and #799. Git is a bloody mess. It's finished, so feel free to review.

### CODE REVIEW

**Comment by Wolvan** [comment]
Glad I could do my part to help out

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -1946,6 +1946,7 @@ XKit.tools.dump_config = function(){
 				if (type === "ok") { m_class = "notification-ok"; }
 				if (type === "error") { m_class = "notification-error"; }
 				if (type === "warning") { m_class = "notification-warning"; }
+				if (type === "pokes") { m_class = "notification-pokes"; }
```
Comment (hobinjk) [inline Extensions/xkit_patches.js]: Should bump version of xkit_patches for this change

File: Extensions/xkit_patches.js [hunk 2]
```diff
@@ -1781,8 +1781,8 @@ XKit.tools.dump_config = function(){
 					m_return.following = true;
 				}
 
-				if ($("body").hasClass("tagged_actions_display") && document.location.href.indexOf("/tagged") !== 1) {
-					m_return.tagged = true;
+				if ($("body").hasClass("tagged_actions_display") && document.location.href.indexOf("/tagged") !== 1) {
+					m_return.tagged = true;
```
Comment (hobinjk) [inline Extensions/xkit_patches.js]: This is unrelated and should not be included

File: Extensions/xkit_patches.css [hunk 3]
```diff
@@ -2,11 +2,15 @@
...
 }
 
+.xkit-notification.notification-pokes {
+	background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAMAAABhEH5lAAAAIVBMVEWwsLAwMDBIUFB4eICYSDigoMDYWDjYyPD4kDj40Dj4+Phh7PQLAAAAAXRSTlMAQObYZgAAAGdJREFUeNp10FsOgCAMRFGZUpHuf8GOUwga4v2CE17hyIrScMKlFgrUWab0vmxIRJ9E20kHvTfSREryS/UrlQQaAIIEDzEPh+bsudKAiEBmelcL46qE5qQ0s6ZSaN4GSNKKq5Kyf+EN9RkFeT+kVbYAAAAASUVORK5CYII=');
+}
+
 #xkit-window.xkit-wide-window,
 #xkit-window-old.xkit-wide-window {
 
-	width: 650px;
```
Comment (hobinjk) [inline Extensions/xkit_patches.css]: These whitespace changes would ideally be in a different PR

**Comment by hobinjk** [comment]
Functionality LGTM. Another minor nitpick is that the official spelling of Pokémon is Pokémon, not Pokemon.

File: Extensions/xkit_patches.css (hunk 3 above)
Comment (Wolvan) [inline Extensions/xkit_patches.css]: The whitespaces may come from the editor automatically

File: Extensions/xkit_patches.js (hunk 2 above)
Comment (Wolvan) [inline Extensions/xkit_patches.js]: What exactly changed here, I don't see it

File: Extensions/pokes.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+			return false;
+		}
+	},
+
+	pokeGen: function() {
+		return Math.floor(Math.random() * 889);
+	},
+
+	destroy: function() {
+		this.running = false;
```
Comment (hobinjk) [inline Extensions/pokes.js]: Don't forget to remove the post listener here.

File: Extensions/xkit_patches.js (hunk 2 above)
Comment (ThePsionic) [inline Extensions/xkit_patches.js]: Neither do I, to be honest.

File: Extensions/xkit_patches.css (hunk 3 above)
Comment (ThePsionic) [inline Extensions/xkit_patches.css]: I'm 100% certain they are.

File: Extensions/xkit_patches.js (hunk 1 above)
Comment (ThePsionic) [inline Extensions/xkit_patches.js]: Done

File: Extensions/pokes.js (hunk 4 above)
Comment (ThePsionic) [inline Extensions/pokes.js]: Done

**Comment by Wolvan** [comment]
@hobinjk can you check again? The PR is updated with the changes

**Comment by hobinjk** [comment]
LGTM
@homu r+
